ios/FluentUI/Card/CardView.swift

```diff
@@ -33,9 +33,9 @@ public enum CardStyle: Int, CaseIterable {
     var width: CGFloat {
         switch self {
         case .horizontal:
-            return 156
+            return 175
         case .vertical:
-            return 120
+            return 97
```

Comment:
Were these intended to be final designs? 97 seems like a weird number.

Reply:
Ill double check with design as this is what it was in the figma.